addon/orm/model.js

```diff
@@ -221,6 +223,7 @@ class Model {
 
     // Define the getter/setter
     Object.defineProperty(this, attr, {
+      configurable: true,
```

Comment:
done, but wouldn't it be dangerous for users trying to set directly the property without using update? would mess things up then right?